apps/system/lockscreen/js/lockscreen_state_manager.js

```diff
@@ -542,10 +564,13 @@
           this.onUnlock();
         }
         break;
+      case 'lockscreen-appopened':
       case 'lockscreen-request-lock':
+        this.resumeEvents(['holdcamera']);
```

Comment:
You can just use `this.resumeEvents('holdcamera')` and save it in the map. To have an array is not necessary for the arguments.